app/vue/src/client/preview/index.ts

```diff
@@ -10,51 +9,47 @@ import {
 } from '@storybook/addons';
 
 import './globals';
-import { IStorybookSection, StoryFnVueReturnType } from './types';
-
-import render, { VALUES } from './render';
+import { VueOptions, IStorybookSection, StoryFnVueReturnType, VALUES, WRAPS } from './types';
+import render from './render';
 import { extractProps } from './util';
 
-export const WRAPS = 'STORYBOOK_WRAPS';
+const isVueConstructor = (component: unknown): component is VueConstructor =>
+  (component as VueConstructor)._isVue; // eslint-disable-line no-underscore-dangle
 
+function prepare(rawStory: undefined, innerStory?: VueConstructor): undefined;
```

Comment:
A little bit awkward, but maintains the type-safety effectively, since `prepare` will only return `undefined` if `rawStory === undefined`, which won't happen if someone uses typescript, but may happen otherwise.